content/docs/introduction/glossary.md

```diff
@@ -108,11 +108,18 @@ write into a format the other system can understand.
 
 A remote write endpoint is what Prometheus talks to when doing a remote write.
 
+### Sample
+
+A sample is the smallest unit of time series data in Prometheus. Each sample consists of:
```

Comment:
It feels odd to call a sample a unit, a sample is one point of a time series.

Reply:
Yeah, I would just say it's a single value at a point in time for a time series or something like that.